wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
This appears to be the same files as the one in the .mvn folder. Do we need them both?

Reply:
Good catch, in fact the wrapper folder should be removed.